### PR TITLE
feat: add comment for expired PRs, fix: add link to note comment

### PR DESCRIPTION
**Pull request by sqin2019**

### CODE REVIEW

File: .github/workflows/expire.yml [hunk 1]
```diff
@@ -86,6 +86,14 @@ jobs:
                   pull_number: pull['number'],
                   state: 'closed',
                 });
+
+                // Comment on closed PRs.
+                await github.rest.issues.createComment({
+                  owner: context.repo.owner,
+                  repo: context.repo.repo,
+                  issue_number: pull['number'],
+                  body: `**\`Access on Demand\`** - request closed due to inactivity for ${{ inputs.expiry_hours }} hours`,
```
Comment (dcreey) [inline .github/workflows/expire.yml:95]: NIT: `request closed due to inactivity for at least ${{ inputs.expiry_hours }} hours`

File: .github/workflows/note.yml [hunk 2]
```diff
@@ -22,14 +22,18 @@ on:
...
+        required: true
 
 env:
   AOD_NOTE: >
     ⛔️ <strong>This is an AOD request, and merging is NOT allowed.</strong>
     The request will be automatically applied once the code owners approve it.
     Please close the PR once you are finished or it will automatically be closed
     after ~${{ inputs.expiry_hours }} hours of it's last update. For more
-    instructions, please see [here](https://github.com/abcxyz/aod-template/blob/main/README.md).
+    instructions, please see [here](${{ inputs.aod_instruction_link }}).
```
Comment (dcreey) [inline .github/workflows/note.yml:36]: Could you default it to the repo root README? This should be discoverable.

File: .github/workflows/expire.yml (hunk 1 above)
Comment (sqin2019) [reply .github/workflows/expire.yml]: Thanks, fixed!

File: .github/workflows/note.yml (hunk 2 above)
Comment (sqin2019) [reply .github/workflows/note.yml:36]: ack

**Review by dcreey — APPROVED** [review]

File: .github/workflows/note.yml (hunk 2 above)
Comment (sqin2019) [reply .github/workflows/note.yml:37]: defaulted to repo root instead just in case readme is not added, at repo root can also see readme.

Comment (sqin2019) [reply .github/workflows/note.yml:37]: closing, feel free to reopen.

**Pull request merged**